types/asynciterator/index.d.ts

```diff
@@ -8,7 +8,7 @@
 
 import { EventEmitter } from "events";
 
-export abstract class AsyncIterator<T> extends NodeJS.EventEmitter {
+export abstract class AsyncIterator<T> extends EventEmitter {
```

Comment:
I think `/// <reference types="node" />` is not needed anymore after this change.

Reply:
Please sure it's still needed, how else would it know where to get `events` imported from?

Reply:
There is a dedicated `events` package (folder types\events\), see https://www.npmjs.com/package/@types/events and https://www.npmjs.com/package/events